<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiskExplorer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/app.ico</normaloff>:/resource/app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="leftTopHLayout">
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AllBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>GetAll</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>298</width>
          <height>312</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionToken"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionSimple_Chinese"/>
    </widget>
    <addaction name="actionSideBar"/>
    <addaction name="menuLanguage"/>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUpload_File"/>
    <addaction name="actionNew_Folder"/>
    <addaction name="actionDelete_Folder"/>
    <addaction name="separator"/>
    <addaction name="actionPrerefrence"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/open.png</normaloff>:/resource/open.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionToken">
   <property name="text">
    <string>Token</string>
   </property>
   <property name="toolTip">
    <string>Token</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/exit.png</normaloff>:/resource/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSideBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SideBar</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionSimple_Chinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simple Chinese</string>
   </property>
  </action>
  <action name="actionUpload_File">
   <property name="text">
    <string>Upload File</string>
   </property>
  </action>
  <action name="actionNew_Folder">
   <property name="text">
    <string>New Folder</string>
   </property>
  </action>
  <action name="actionDelete_Folder">
   <property name="text">
    <string>Delete Folder</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionPrerefrence">
   <property name="text">
    <string>Prerefrence</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionConnect</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectToDisk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToken</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeToken()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AllBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>getAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSideBar</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>switchSideBar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_Folder</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUpload_File</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>uploadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete_Folder</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRefresh</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrerefrence</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>prerefrence()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connectToDisk()</slot>
  <slot>changeToken()</slot>
  <slot>exit()</slot>
  <slot>getAll()</slot>
  <slot>switchSideBar()</slot>
  <slot>about()</slot>
  <slot>switchLang(QAction*)</slot>
  <slot>uploadFile()</slot>
  <slot>newFolder()</slot>
  <slot>deleteFolder()</slot>
  <slot>refreshFolder()</slot>
  <slot>prerefrence()</slot>
 </slots>
</ui>
